run_service.sh

```diff
@@ -10,6 +10,12 @@ then
 fi
 
 # Check dependencies
+if command -v python3 >/dev/null 2>&1; then
+    PYTHON_CMD="python3"
+else
+    PYTHON_CMD="python"
+fi
```

Comment:
Let's perform the check for `python` too.

Reply:
Addressed